lambdas/answer/src/test/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionAnswerHandlerTest.java

```diff
@@ -162,7 +162,7 @@ void shouldReturn200WithFinalResponseFromExperianAPI() throws IOException, SqsEx
         verify(mockSessionService).createAuthorizationCode(mockSessionItem);
         verify(mockAuditService)
                 .sendAuditEvent(
-                        eq(AuditEventType.THIRD_PARTY_REQUEST_ENDED),
+                        eq(AuditEventType.RESPONSE_RECEIVED),
```

Comment:
Please could you still have the verify on`THIRD_PARTY_REQUEST_ENDED` in addition to the one on RESPONSE_RECEIVED